iocore/net/NetAcceptEventIO.cc

```diff
@@ -0,0 +1,44 @@
+/** @file
+
+  A brief file description
+
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+
+#include "NetAcceptEventIO.h"
+#include "P_NetAccept.h"
+
+int
+NetAcceptEventIO::start(EventLoop l, NetAccept *na, int events)
+{
+  _na = na;
+  return start_common(l, _na->server.fd, events);
+}
+void
+NetAcceptEventIO::process_event(int flags)
+{
+  this_ethread()->schedule_imm(_na);
+}
+
+int
+NetAcceptEventIO::close()
+{
+  EventIO::stop(); // discard retval
```

Comment:
The behavior is slightly changed. This does not return 0 if `!this->syscall`.

Also, if all subclasses have to call base class's close, maybe we can do like:
```
class Base {
public:
  close();
protected:
  virtual _close() = 0;
}

Base::close()
{
  if (!this->syscall) {
    return 0;
  }
  stop();
  _close();
}

Sub1::_close()
{
  // Do sub class specific things
}
```

Reply:
It turns out that `close()` isn't used anywhere.  I've removed it to reduce confusion.